v2/stacks/auth/README.md

```diff
@@ -36,3 +36,35 @@ To get the aws values from the dashboard do the following:
 - copy the highlighted values
 
 Note that the AWS_SESSION_TOKEN is only valid for 12 hours. Once the token has expired you would need to stop the stack, retrieve and set new credentials before running the stack again.
+
+# Example on how to debug a Go application running in docker 
```

Comment:
I'd be tempted to put this in the parent directory's `README.md` because it's not specific to `auth`

Reply:
This is now addressed.